ArduPlane/mode.cpp

```diff
@@ -101,6 +101,18 @@ bool Mode::enter()
 
         // update RC failsafe, as mode change may have necessitated changing the failsafe throttle
         plane.control_failsafe();
+
+#if HAL_QUADPLANE_ENABLED
+        if (quadplane.available()) {
```

Comment:
I prefer the way this was originally always set with:
```
attitude_control->set_roll_pitch_rate_tc(quadplane.acro_rp_rate_tc.get());
attitude_control->set_yaw_rate_tc(quadplane.yaw_rate_tc.get());
```
In mode enter:

https://github.com/ArduPilot/ardupilot/blob/902b895aef794c3eac685f24c96a24e15ea724a6/ArduPlane/quadplane.cpp#L4065

And then override with `attitude_control->set_yaw_rate_tc(quadplane.acro_y_rate_tc.get());` in acro enter which is called afterwards. 

https://github.com/ArduPilot/ardupilot/blob/902b895aef794c3eac685f24c96a24e15ea724a6/ArduPlane/mode_qacro.cpp#L6

Reply:
I didn't understand what you meant before but I do now.  I suppose that is more efficient?

Reply:
Just tidier, keeps the mode specific code in the dedicated mode files. Otherwise if were not careful this global mode enter function becomes a maze of if statements.